### PR TITLE
Try to retreive multiple indices for multiple labels

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

**Comment by fppt** [comment]
My own test `vertexUniqueIndexLookupWithMultipleLabels` is still failing. When multiple labels are defined it fails to use the index. I am not entirely sure why so any hints would be very much appreciated.

**Comment by fppt** [comment]
The error was a poorly defined test on my part. I think this is working now.

File: driver/src/test/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphIndexTest.java [hunk 1]
```diff
@@ -106,26 +113,67 @@ public void vertexUniqueIndexLookupWithValue() {
...
+        Vertex v1 = graph.addVertex(label, label1, key, value1);
+        Vertex v2 = graph.addVertex(label, label2, key, value1);
+        Vertex v3 = graph.addVertex(label, label3, key, value1);
+
+        GraphTraversal<Vertex, Vertex> traversal = graph.traversal().V().hasLabel(label1, label2, label3).has(key, value1);
+
+        Set<OrientIndexQuery> indicies = findUsedIndex(traversal);
+        Assert.assertEquals(3, indicies.size());
+
+        assertTrue(valueFound(indicies, v1, value1));
```
Comment (mpollmeier) [inline driver/src/test/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraphIndexTest.java]: we would we get better test assertion errors if you use `assertEquals`.. you could e.g. rename `valueFound` to `assertValueFound` and use `assertEquals` in there. 

just a thought..

**Comment by fppt** [comment]
Thanks for merging Michael. Sorry for not replying sooner, I have been away over the past week. I agree with your comment. It would make the test more informative. In my next PR I will make that minor change.

**Comment by fppt** [comment]
On a different note. When do you think you will make the next release for this plugin ?

**Comment by mpollmeier** [comment]
just released 3.2.1.2 :)

**Comment by fppt** [comment]
Awesome ! Thank you :)

**Pull request merged**